python/run-tests.py

```diff
@@ -93,17 +93,18 @@ def run_individual_python_test(target_dir, test_name, pyspark_python):
         "pyspark-shell"
     ]
     env["PYSPARK_SUBMIT_ARGS"] = " ".join(spark_args)
-
-    LOGGER.info("Starting test(%s): %s", pyspark_python, test_name)
+    str_test_name = " ".join(test_name)
+    LOGGER.info("Starting test(%s): %s", pyspark_python, str_test_name)
     start_time = time.time()
     try:
         per_test_output = tempfile.TemporaryFile()
         retcode = subprocess.Popen(
-            [os.path.join(SPARK_HOME, "bin/pyspark"), test_name],
```

Comment:
Just a thought, could you leave `test_name` as a string and then change this line to `[os.path.join(SPARK_HOME, "bin/pyspark")] + test_name.split(),`?  I think it would be a little more simple and wouldn't need `str_test_name`, wdyt?

Reply:
Oh, yea. Looks that's going to reduce the diff. Let me try.